### DISCUSSION TITLE
Block info/warning...

### DISCUSSION THREAD

**Discussion by binerf** [requarks/wiki#3810] [Help / Questions]
Hi all,

New on wiki.js, I don't understand how you create Info block or Warning block in a page like blocks in documentation.

Ex:
![image](https://user-images.githubusercontent.com/10645373/114201643-b94cfe00-9956-11eb-918e-4c8678bf3d37.png)
Thank you.

**Comment by adenoz** [comment]
Hi, when using the markdown editor, you hit the 'T' in the square. That will drop down where you can select from a range of block options including info and warning etc.
<img width="589" alt="image" src="https://user-images.githubusercontent.com/80598992/114209875-2f119380-9970-11eb-982c-fff11b8412e5.png">
You can also add things like `{.is-info}` for info boxes at the end of any blockquote `>`. Such as:
```
> This is an important thing.{is-info}
```
Which will render as:
<img width="680" alt="image" src="https://user-images.githubusercontent.com/80598992/114210398-cb3b9a80-9970-11eb-899c-8492a4d7d2e1.png">
Hope this helps.

**Reply by binerf** [reply]
Thank you !
I cannot find this icon in visual editor... Did I miss something ?

**Reply by binerf** [reply]
BTW, it seems it doesn't work with a list inside the block:

```
> Contact list:
> - contact 1
> - contact 2
{.is-info}
```

![image](https://user-images.githubusercontent.com/10645373/114213279-1484ed80-9963-11eb-9864-61b456a89c5a.png)

**Reply by adenoz** [reply]
Yeah that is a known bug https://github.com/Requarks/wiki/issues/1241

**Reply by binerf** [reply]
Oh, thanks ! 
Hope it will be fixed in the next release...

**Comment by binerf** [comment]
And to complete, is it possible to create more block types (like "question", "error", etc...) ?

**Reply by adenoz** [reply]
I don't think so. I don't know if this is coming in v3.0 or not. Otherwise you could submit a feature request.

**Reply by binerf** [reply]
Let's go for a feature request.
Thanks !

**Comment by binerf** [comment]
Is it possible to use block types ({.is-info}, {.is-success},...) with the WYSIWYG interface ?
